### ISSUE TITLE
Dual stroke input (from "English (US, alt. intl.)" keybaord layout) not working anymore

### ISSUE THREAD

**Issue by notorand-it** [ungoogled-software/ungoogled-chromium-archlinux#195]
Since 105.0.5195.52-1, input layout "English (US, alt. intl.)" doesn't allow me to enter any dual-stroke character, nor any dual-stroke accented character.
I think I could do that with 104.0.5112.101-1 but I am not sure how to install it back.

Any other software on my system does. Which is:
Chromium: 105.0.5195.52
Plasma Version : 5.25.4-1
KDE Version : 22.08.0-1
Frameworks Version : 5.97.0-1
Qt5 Version : 5.15.5+kde+r4-1
Wayland Version : 1.21.0-1
Mesa Version : 22.1.7-1

For example, to enter singe-quote (') I would normally hit its key (below {) and then space.
Now I only get space, as if I never hit single-quote.
This is happening whenever I need to type, from URL bar to settings to web forms on pages. 

To enter acute-accent-o (ó) I would hit single-quote and then "o" on any other application.
Now I get just "o" in ungoogled-chromium.

Affected characters are those that require two-strokes in this layout:
back-quote,
tilde (both on left of 1), 
caret (on 6), 
single-quote and 
double-quote.

None of these characters can be entered.
To type this comment I had to use copy-paste from terminal!

As a workaround I just discovered that I can use ALT+ with those characters.
Accented vowels cannot be entered at all.

**Comment by PF4Public** [comment]
> Accented vowels cannot be entered at all.

óäï½ — OK if using ALT

Do dual-stroke characters work in other applications? I'm afraid it must be a Chromium issue.

**Comment by notorand-it** [comment]
> > Accented vowels cannot be entered at all.
> 
> óäï½ — OK if using ALT
> 
> Do dual-stroke characters work in other applications? I'm afraid it must be a Chromium issue as they are known to break things: [ungoogled-software/ungoogled-chromium#2052](https://github.com/ungoogled-software/ungoogled-chromium/issues/2052).

Yes, it is a chromium issue and its wayland support which is seems it changed from v104 to v105.
By removing `--enable-features=UseOzonePlatform --ozone-platform=wayland` command line arguments from the launcher I got the expected behavior back.
Sorry for bothering: maybe this workaround can help someone.
